src/psyclone/tests/psyir/nodes/otter_nodes_test.py

```diff
@@ -186,10 +157,7 @@ def test_otterloopnode_lower_to_language():
     calls = routine.walk(Call)
     assert len(calls) == 2
     assert calls[0].routine.name == "fortran_otterLoopBegin_i"
-    assert len(calls[0].children) == 3
-    assert calls[0].children[0].name == "__FILE__"
-    assert calls[0].children[1].value == "my_routine"
-    assert calls[0].children[2].name == "__LINE__"
+    assert len(calls[0].children) == 0
```

Comment:
Hi Adam,

Not sure whats happened here but this test is broken - looks like you changed some stuff here to do with the calls for the argument, might wanted to revert these lines.